ops/travis/unit-testing/deployment-agent

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -e
+
+FOLDER_CHECKS="deployment-agent\/"
+
+before_install() {
+    if [ -n "$(bash ops/travis/helpers/test_for_changes deployment-agent)" ];
```

Comment:
using the aproach I added in my simcore PR seems nicer to me :)

Reply:
done